lib/bolognese/writers/jats_writer.rb

```diff
@@ -76,7 +76,9 @@ def insert_citation_title(xml)
       end
 
       def insert_source(xml)
```

Comment:
Please note, this change is to fix a spec, which started to fail, because the Jats writer uses the `crossref_reader`.